WebAPI/ViewModels/Timetable.cs

```diff
@@ -1,9 +1,9 @@
-using System.Collections.Generic;
-
-namespace WebAPI.Models
-{
-    public class Timetable
-    {
-        public List<TimetableBlock> Blocks { get; set; }
-    }
-}
+using System.Collections.Generic;
+
+namespace WebAPI.Models
+{
+    public class Timetable
+    {
+        public List<TimetableBlock> Blocks { get; set; }
+    }
+}
```

Comment:
Lot's of changes, which are not real changes... Not just here. You should revert them...

Reply:
I move some files so probably that's why the changes are shown.